getting-started/deployment/aws-ec2.mdx

```diff
@@ -87,11 +87,100 @@ After making changes, restart the Docker Compose containers:
 sudo docker compose down && sudo docker compose up -d
 ```
 
-## TLS
+## Enabling TLS (HTTPS) access with a load balancer
 
-For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance.
+For production usage, it's recommended to enable TLS (HTTPS). Without TLS, cloud templates may not load properly when using the Safari browser.
 
-Without TLS, cloud templates may not load properly when using the Safari browser.
+To secure OpenOps with TLS, it's recommended to place your EC2 instance behind an AWS Application Load Balancer (ALB). This setup uses AWS Certificate Manager (ACM) to handle TLS, and allows secure access via a DNS name, even without a [custom domain](#accessing-openops-via-a-custom-domain-over-tls).
+
+<Warning>Once the ALB is in place, all access should go through it — you can no longer use the EC2 public IP directly.</Warning>
+
+### Request a TLS certificate in ACM
+
+1. Go to the [AWS Certificate Manager](https://console.aws.amazon.com/acm/home) (ACM).
+2. Request a public certificate for your EC2's public DNS name (e.g., `ec2-1-2-3-4.compute-1.amazonaws.com`).
```

Comment:
that won't work, you need a custom domain.

https://stackoverflow.com/a/66687070/24956286